src/screens/Learn/Learn.style.tsx

```diff
@@ -51,11 +47,14 @@ export const BreadcrumbsContainer = styled.div`
 export const BodyCopyStyles = css`
   color: ${COLOR_MAP.GRAY_BODY_COPY};
   line-height: 1.4;
-  letter-spacing: 1;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 22px;
```

Comment:
same set of comments re: these styles written above for `MarkdownBody`. an overall suggestion: maybe update/clean them up here in `BodyCopyStyles` and import `BodyCopyStyles` into `Accordion.style.tsx` and use them to declare styles for `MarkdownBody`, so we only need to edit them in one place going forward